### DISCUSSION TITLE
BT-1701 Winner Organisation Identifier part of next release?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#225] [Q&A] [answered]
Hi,
We can see a new identifier, BT-1701 Winner Organisation Identifier, in the SDK documentation for Withheld publication of information. But it is not part of any other specifications yet. What are the plans for this BT? Will that be clarified and identified in the next release?

From SDK 1.3.2
![image](https://user-images.githubusercontent.com/79446518/203497767-fb403273-0e67-43c7-895d-125c282d7100.png)
![image](https://user-images.githubusercontent.com/79446518/203498019-f2efd513-85c8-470e-822f-1aad8ecd093f.png)

Kind regards,
Åsa

**Comment by YvesJo** [comment, marked as answer]
BT-1701 is not a new concept, it was already present in documentation of SDK 1.1.1; it was developed and discussed during the eForms Consultation and prior to the Regulation publication.
This BT does not however exist in the Regulation while the possibility of hiding that information may still be required.
The mechanism for withholding Winner details is more complex than the one for other information as it involves multiple concepts and is not just about hiding/removing a single reference. This will be shared in a next release.
